src/util.js

```diff
@@ -91,3 +91,10 @@ export const writeTextToClipboard = (string: string) => {
     write();
   }
 };
+
+export const resolveFalsyValues = (value: string | number | boolean | null) => {
```

Comment:
What is this?

Reply:
I added this as part of the flow of `getValue({})`.
I found an edge case where `value === 0` and instead of showing '0' it shows `""`

so I added this as inner part of `getValue()`

Reply:
I think you mean getEmptyValue

Reply:
This is the full fn:


```typescript
export const resolveFalsyValues = (value: string | number | boolean | null) => {
  if (typeof value === "boolean") return value;
  if (value === 0) return 0;
  if (value === "") return "";
  return value;
};
```

Actually its more of a `resolveValue` then `resolveFalsyValues` cuz it return the value in the end. Ist addition is that it just knows how to deal with a falsy value.

I didnt want to call it `resolveValue` cuz its a bit confusion calling a `resolveValue` under a `getValue()` .